src/app/shared/actions/account-actions/actions/account-delete-action.ts

```diff
@@ -1,15 +1,13 @@
 import { Injectable } from '@angular/core';
-import { Observable } from 'rxjs/Observable';
 import { JobsNotificationService } from '../../../services/jobs-notification.service';
 import { BaseAccountAction } from './base-account-action';
 import { DialogService } from '../../../../dialog/dialog-service/dialog.service';
-import { AccountService } from '../../../services/account.service';
-import { Account } from '../../../models/account.model';
 
 
 @Injectable()
 export class AccountDeleteAction extends BaseAccountAction {
```

Comment:
Do we need this Action classes? can we add just some constant Action interface and export it?